Packs/FeedUnit42v2/Integrations/FeedUnit42v2/FeedUnit42v2.py

```diff
@@ -311,7 +321,13 @@ def get_attack_id_and_value_from_name(attack_indicator):
     indicator value = 'Redundant Access'.
     """
     ind_name = attack_indicator.get('name')
-    idx = ind_name.index(':')
+    separator = ':'
+    try:
+        idx = ind_name.index(separator)
+    except ValueError as err:
+        error_msg = f"Could not find separator {separator} in attack indicator {ind_name}, error: {err}"
+        demisto.debug(error_msg)
+        raise DemistoException(error_msg)
```

Comment:
No need for both, when you raise the error it will be matched in the main function and printed to logs by the return_error.

Reply:
You are right, ill remove the debugging